tests/test_integration.py

```diff
@@ -74,8 +97,75 @@ def test_can_delete_reservation(self):
         self.client.force_authenticate(self.user)
         response = self.client.delete(
             f"/api/v1/reservation/delete/{self.reservation.id}/")
+        print(response.data)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
 
+    def test_create_reservation_with_existing_time(self):
+        """
+        In test database there is room with this time:
+        date_from: 2021-08-07T19:57:01.615Z
+        date_out: 2021-09-07T19:57:01.615Z
+        """
```

Comment:
but we've also discussed to create a test with reservation time, which wraps the existing event, right? Not all cases covered by tests